samples/subsys/ipc/ipm_mcux/README.rst

```diff
@@ -0,0 +1,138 @@
+.. _ipm-mcux-sample:
```

Comment:
Since this is a new documentation area in the samples, you'll need to create an index/TOC page named ``samples/subsys/ipc/ipc.rst`` with the contents:

    .. _ipc_samples:

    IPC Samples
    ###########

    .. toctree::
       :maxdepth: 1
       :glob:

       **/*

(This will fix the "not included in any TOC" error.)

Reply:
Added. Is there a way to validate the rst docs myself before pushing? Thanks.

Reply:
Thanks.